openslides/motions/views.py

```diff
@@ -395,9 +395,9 @@ def destroy(self, *args, **kwargs):
         """
         Customized view endpoint to delete a motion poll.
         """
-        result = super().destroy(*args, **kwargs)
         poll = self.get_object()
         poll.motion.write_log([ugettext_noop('Vote deleted')], self.request.user)
+        result = super().destroy(*args, **kwargs)
```

Comment:
Please do not write the log before the poll was deleted. Maybe this line has to be one line earlier.